content/en/watchdog/insights.md

```diff
@@ -249,6 +250,23 @@ In the full side panel, you can see a timeseries graph about the performance met
 [1]: /real_user_monitoring/explorer/search/#facets
 [2]: /real_user_monitoring/browser/monitoring_page_performance/#event-timings-and-core-web-vitals
 [3]: /real_user_monitoring/browser/monitoring_page_performance/#monitoring-single-page-applications-spa
+{{% /tab %}}
+{{% tab "Synthetic Monitoring" %}}
+
+### Error outliers
+
+Error outliers in Synthetic Monitoring display unexpected behaviors and performance deviations. These anomalies provide insights into the reliability issues in your [Synthetic Browser Tests][101]. Identifying these error outliers helps you troubleshoot errors in failed test runs, enhancing debugging and reducing Mean Time To Resolution (MTTR).
+
+When reviewing failed test runs, you can see the number of error outliers on the failed test:
+
+{{< img src="watchdog/synthetics_watchdog_outlier.png" alt="An overview of a failed browser test run with test step details and the error message which was identified as an error outlier for a failing test step by Watchdog" style="width:100%;" >}}
+
+Click on the outlier from the above step to access the error outlier message on the **Errors & Warnings** tab in the test step side panel.
```

Comment:
Totally optional, but I like to have sentences like this in sequence (for example, users see the panel before they see the tab, so I like to mention them in that order). I think it helps cut down the cognitive load it takes for a user to follow the docs. But again, I wouldn't see it as a blocker here.
```suggestion
To view the error outlier message, click on the outlier. Then, on the test step side panel, click the **Errors & Warnings** tab.
```